libs/sql-schema-describer/src/ids.rs

```diff
@@ -0,0 +1,39 @@
+use crate::{Column, Enum, SqlSchema, Table};
+use std::ops::Index;
+
+/// The identifier for a table in a SqlSchema. Use it with the indexing syntax:
+/// `let table = schema[table_id];`
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct TableId(pub(crate) u32);
```

Comment:
Would even a u16 suffice here?

Reply:
65,536 tables or columns is unlikely but possible — u32 is already a win. We could tune that easily later.